### PR TITLE
fix: Workflow always print "error:" even if there are none

### PR DESCRIPTION
**Pull request by matteovava**
Fix issue [https://github.com/medusajs/medusa/issues/8976](https://github.com/medusajs/medusa/issues/8976)

### CODE REVIEW

File: packages/core/workflows-sdk/src/helper/workflow-export.ts [hunk 1]
```diff
@@ -527,7 +527,7 @@ function attachOnFinishReleaseEvents(
         )
         .join(EOL + separator + EOL)
 
-      logger.error(allWorkflowErrors)
+      if (allWorkflowErrors) logger.error(allWorkflowErrors)
```
Comment (sradevski) [inline packages/core/workflows-sdk/src/helper/workflow-export.ts]:
```suggestion
      if (allWorkflowErrors) {
        logger.error(allWorkflowErrors)
      }
```

**Review by sradevski — COMMENTED** [review]
Nice, thanks!  Let's just reformat it to be more consistent (we need to check why our linting/prettier setup didn't catch this)

File: packages/core/workflows-sdk/src/helper/workflow-export.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/core/workflows-sdk/src/helper/workflow-export.ts]: Should we maybe just commit this ourselves and get it merged?

Comment (sradevski) [reply packages/core/workflows-sdk/src/helper/workflow-export.ts]: Let's merge it, I'll reformat in a separate PR

Comment (matteovava) [reply packages/core/workflows-sdk/src/helper/workflow-export.ts]: Hi, sorry i've seen it only now and accepted the commit suggestion

**Review by carlos-r-l-rodrigues — APPROVED** [review]
Thanks @matteoxplo

**Pull request merged**